test/system/010-images.bats

```diff
@@ -330,6 +296,7 @@ Deleted: $pauseID"
     run_podman image rm --force bogus
     is "$output" "" "Should print no output"
 
+    run_podman pull $IMAGE
```

Comment:
never pull $IMAGE in system tests, pull always pulls and thus can cause network flakes.
use `_prefetch $IMAGE`